Week-23/todo-list/.env

```diff
@@ -0,0 +1 @@
+db_url='mongodb+srv://Local:<password>@local.fic77ty.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Add this file to .gitignore, you don’t want leaking secret info to public git repos.
Also, env var naming should be all capitals, snake cased, for eg, it should be DATABASE_URL instead of database_url.